lua/nvim_aider/nvim_tree.lua

```diff
@@ -60,20 +61,17 @@ end
 
 ---@param opts? nvim_aider.Config
 function M.setup(opts)
-  local ok, _ = pcall(require, "nvim-tree")
-  if ok then
-    vim.api.nvim_create_user_command("AiderTreeAddReadOnlyFile", add_read_only_file_from_tree, {
-      desc = "Add read-only file from nvim-tree to Aider chat",
-    })
-
-    vim.api.nvim_create_user_command("AiderTreeAddFile", add_file_from_tree, {
-      desc = "Add file from nvim-tree to Aider chat",
-    })
-
-    vim.api.nvim_create_user_command("AiderTreeDropFile", drop_file_from_tree, {
-      desc = "Drop file from nvim-tree from Aider chat",
-    })
-  end
+  vim.api.nvim_create_user_command("AiderTreeAddReadOnlyFile", add_read_only_file_from_tree, {
+    desc = "Add read-only file from nvim-tree to Aider chat",
+  })
+
+  vim.api.nvim_create_user_command("AiderTreeAddFile", add_file_from_tree, {
+    desc = "Add file from nvim-tree to Aider chat",
+  })
+
+  vim.api.nvim_create_user_command("AiderTreeDropFile", drop_file_from_tree, {
+    desc = "Drop file from nvim-tree from Aider chat",
+  })
```

Comment:
this removes the optional requirement `nvim-tree`and make it mandatory. because the plugin will crash if not installed.